### PR TITLE
MHV 50758 - SM landing page unread count loading

### PR DESCRIPTION
**Pull request by alex-morgun-1**
## Summary

Context: in Philly usability testing, we saw that this would load last on the page and confused users because it's the key CTAs on the page.

User Story: As a Secure Messaging User, I want to see the "Go to inbox" and "Start a new message" load first, so I am not confused when my page takes a long time to load

> GIVEN: user lands on SM landing page
> WHEN: Page takes a long time to load
> THEN: "Go to inbox" and "Start a new message" load first

## Related issue(s)

-  https://jira.devops.va.gov/browse/MHV-50758

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|--------| Before | After |
|Desktop|||

 ## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Review by vsaleem — APPROVED** [review]
Looks great!✅

**Review by Asterix22 — APPROVED** [review]

File: src/applications/mhv/secure-messaging/tests/e2e/pages/SecureMessagingLandingPage.js [hunk 1]
```diff
@@ -36,6 +36,33 @@ class SecureMessagingLandingPage {
       .should('be.visible')
       .and('have.text', `${text}`);
   };
+
+  verifyUnreadMessagesNote = (text = 'unread messages in your inbox') => {
+    cy.get('.vads-u-font-size--h3')
```
Comment (Asterix22) [inline src/applications/mhv/secure-messaging/tests/e2e/pages/SecureMessagingLandingPage.js:41]: If possible, change from selecting on styling attribute

**Review by Asterix22 — CHANGES_REQUESTED** [review]

File: src/applications/mhv/secure-messaging/tests/e2e/pages/SecureMessagingLandingPage.js (hunk 1 above)
Comment (fazilqa) [reply src/applications/mhv/secure-messaging/tests/e2e/pages/SecureMessagingLandingPage.js:41]: agree, will ask devs to add more data-testid attributes

Comment (fazilqa) [reply src/applications/mhv/secure-messaging/tests/e2e/pages/SecureMessagingLandingPage.js]: requested changes done

**Review by Asterix22 — APPROVED** [review]

**Review by Asterix22 — APPROVED** [review]

**Pull request merged**